pom.xml

```diff
@@ -40,6 +40,13 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>io.jenkins.tools.bom</groupId>
+          <artifactId>bom-2.401.x</artifactId>
+          <version>2329.v078520e55c19</version>
+          <type>pom</type>
+          <scope>import</scope>
+      </dependency>
```

Comment:
Why?  we do not depend on any other plugin - this would seemingly be superfluous?